cmd/rigs/cmd/publish_update_deals.go

```diff
@@ -0,0 +1,30 @@
+package cmd
+
+import (
+	"time"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"golang.org/x/time/rate"
+)
+
+func init() {
+	updateDealsCmd.Flags().Int("concurrency", 10, "number of concurrent requests to car storage service")
+	updateDealsCmd.Flags().Duration("rate-limit", time.Millisecond*350, "rate limit for requests to car storage service")
+	updateDealsCmd.Flags().StringSlice("ids", []string{}, "limit to rigs with these ids")
+
+	publishCmd.AddCommand(updateDealsCmd)
+}
+
+var updateDealsCmd = &cobra.Command{
```

Comment:
New command that pulls the latest storage status from web3/nft.storage and stores the latest state in `local.db`. You can optionally provide a list of rig ids if you want to limit the rigs to check.